mlflow/projects/__init__.py

```diff
@@ -619,7 +619,7 @@ def _get_conda_command(conda_env_name):
         if os.name != "nt":
             return ["source %s %s 1>&2" % (activate_path, conda_env_name)]
         else:
-            return ["conda activate %s" % (conda_env_name)]
+            return ["conda.bat activate %s" % (conda_env_name)]
```

Comment:
Is it `conda.exe` or `conda.bat`?

cc: @tomasatdatabricks @dbczumar

Reply:
Quick bump on this

Reply:
What's the background for this. Have you tested it on windows @juntai-zheng ? Can you add a windows test?